src/test/java/org/opentripplanner/routing/algorithm/mapping/__snapshots__/ElevationSnapshotTest.snap

```diff
@@ -2891,7 +2891,7 @@ org.opentripplanner.routing.algorithm.mapping.ElevationSnapshotTest.directWalk=[
               "area": false,
               "bogusName": false,
               "distance": 135.08,
-              "elevation": "0,55.5,65,55.5,65,55.5,120,55.5,120,55.5,135,55.5",
```

Comment:
Some of these elevation changes that are now removed were clear repetitition but this for example now omits some values completely and I don't quite understand if this is intended or not.

Reply:
We merged two ways of concatenating lines, and seems that it was also used here. Previously we aded both the last and first coordinates even if they were equal, now it skips those if those are equal.

Reply:
Thank you for spotting this. I will take a look and see way it is removed - I see all y values are the same, but at least the first and the last point should be there.

Reply:
I will check it in the next PR.

Reply:
There is no error here, the first and last coordinate is kept:

`(0, 55.5), (65, 55.5), (65, 55.5), (120, 55.5), (120, 55.5), (135, 55.5)` -> `(0, 55.5), (135, 55.5)`

This is a horizontal line, so keeping only the first and last coordinate is fine. I though the entire profile was deleted, but it is not.